app/api/events.py

```diff
@@ -183,6 +184,9 @@ def is_joined_event(request: Request, eid:str, token: AccessTokenPayload = Depen
     # uses event["eid"] instead of casting eid -> UUID 
     user = db.events.find_one({"eid" : event["eid"]}, {"participants" : {"$elemMatch" : {"id": token.user_id}}})
 
+    if not user:
```

Comment:
I don't think this is necessary as the KeyError catches this. If we want a separate check we should return 400

Reply:
We can return 400, but KeyError does not handle this. `find_one` returns None if not present, and then it will be a `TypeError` since None object is not subscriptable

Reply:
i see, 400 seems like a suitable response